scripts/RAG.py

```diff
@@ -3,21 +3,22 @@
 import ollama  # Ensure the Ollama library is imported
 import TextExtractor as te
 import VectorDB as vdb
+import yaml
+import sys
 
 class RAG:
-    def __init__(self, input_dir: str, output_dir: str, chroma_db_dir: str, chroma_db_name: str, model="mxbai-embed-large"):
-        self.vector_db = self._setup_vector_db(input_dir, output_dir, chroma_db_dir, chroma_db_name, model)
+    def __init__(self, chroma_db_dir: str, chroma_db_name: str, model="mxbai-embed-large"):
+        self.vector_db = self._setup_vector_db(chroma_db_dir, chroma_db_name, model)
         logging.basicConfig(level=logging.INFO)
     
-    def _setup_vector_db(self, input_dir, output_dir, chroma_db_dir, chroma_db_name, model):
+    def _setup_vector_db(self, chroma_db_dir, chroma_db_name, model):
         """Check if the database exists and set up if not."""
         try:
             # Attempt to load existing vector database
-            vector_db = vdb.VectorDB(input_dir, output_dir, chroma_db_dir, chroma_db_name, model)
+            vector_db = vdb.VectorDB(chroma_db_dir, chroma_db_name, model)
             # Check if the database is empty or needs updating
             if not self._is_database_populated(vector_db):
-                logging.info("Database is not populated. Loading data...")
-                vector_db.load_data()
+                print("Database is not populated. Be sure to load data before querying.")
```

Comment:
This message is OK, but instead of a print it should be a logging.warning() instead.